libs/event/src/lib/pages/analytics/analytics.component.ts

```diff
@@ -135,4 +139,138 @@ export class AnalyticsComponent implements OnInit {
     return start.getTime() < Date.now();
   }
 
+  // Create Event Statistic Excel
+  private async exportExcelFile() {
+    let staticticsTitle: string;
+    if (this.event.type === 'screening') {
+      const titleId = (this.event.meta as Screening).titleId;
+      const { title } = await this.movieService.getValue(titleId);
+      staticticsTitle = title.international;
+    }
+    else if (this.event.type === 'slate') staticticsTitle = this.event.title;
```

Comment:
since we only handle slate & screening event type, you can do :
```suggestion
    let staticticsTitle = this.event.title;
    if (this.event.type === 'screening') {
      const titleId = (this.event.meta as Screening).titleId;
      const { title } = await this.movieService.getValue(titleId);
      staticticsTitle = title.international;
    }
```